aikau/src/test/resources/alfresco/layout/AlfStackContainerTest.js

```diff
@@ -0,0 +1,418 @@
+/**
+ * Copyright (C) 2005-2015 Alfresco Software Limited.
+ *
+ * This file is part of Alfresco
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * This is a unit test for AlfStackContainer
+ * 
+ * @author Richard Smith
+ */
+define(["intern!object",
+        "intern/chai!expect",
+        "intern/chai!assert",
+        "require",
+        "alfresco/TestCommon",
+        "intern/dojo/node!leadfoot/keys"], 
+        function (registerSuite, expect, assert, require, TestCommon, keys) {
+
+   var browser;
+   registerSuite({
+      name: "Stack Container Tests",
+
+      setup: function() {
+         browser = this.remote;
+         return TestCommon.loadTestWebScript(this.remote, "/AlfStackContainer", "Stack Container Tests").end();
+      },
+
+      beforeEach: function() {
+         browser.end();
+      },
+
+      "Check the stack container is present": function () {
+         return browser.findByCssSelector("div.alfresco-layout-AlfStackContainer > div.dijitStackContainer")
+            .then(function () {
+               
+            },
+            function () {
+               assert(false, "Stack container not rendered");
+            });
+      },
+
+      "Check the correct number of panes is present": function() {
+         return browser.findAllByCssSelector("div.dijitStackContainer > div.dijitStackContainerChildWrapper")
+            .then(
+               function (panels) {
+                  expect(panels.length).to.equal(5, "There are not 5 panes");
```

Comment:
Use `assert` only, not `expect`, and do not repeat the error condition in the error message.

e.g. `assert.length(panels, 5, "Incorrect number of initial panes");`